client/src/component/signin/index.css

```diff
@@ -2,8 +2,6 @@
     margin: 0;
     padding: 0px;
     box-sizing: border-box;
```

Comment:
Name it style.css, and why there is a style file out of the component folder? let us divide our app into components and pages folders and in each component, we can add the style and the JSX files.